frontend/app/views/fragments/page/sectionCardStack.scala.html

```diff
@@ -1,4 +1,4 @@
-@(img: com.gu.memsub.images.ResponsiveImageGroup, leadIn: Option[String] = None)(content: Html)
+@(img: com.gu.memsub.images.ResponsiveImageGroup, leadIn: Option[String] = None, topPadding : Boolean = true)(content: Html)
```

Comment:
With a wider fragment refactor it might be possible to avoid this but this seems ok for now.